docs/api.md

```diff
@@ -70,6 +70,7 @@ These are the API group versions which are curently available when using Antrea.
 | CRD | CRD version | Introduced in | Deprecated in | Removed in |
 |---|---|---|---|---|
 | `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v1.12.0 [^1] |
+| `Tier` | v1alpha1 | v1.0.0 | v1.13.0 | N/A |
```

Comment:
You can update `v2.0.0` in `Removed in ` as @antoninbas  suggested.

Reply:
Done.